src/main/java/ListUtils.java

```diff
@@ -0,0 +1,244 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.BufferedWriter;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class ListUtils {
+
+    //define filepath
+    private static final String FILE_PATH = "./data/quag.txt";
+    // create list of tasks
+    private static List<Task> userInputs = new ArrayList<>();
+
+    //class to store task
+    public static class Task {
+        String description;
+        boolean isDone;
+        Task(String description) {
+            this.description = description;
+            this.isDone = false;
+        }
+        void markAsDone() {
+            this.isDone = true;
+        }
+
+        void markAsNotDone() {
+            this.isDone = false;
+        }
+
+        @Override
+        public String toString() {
+            return "[" + (isDone ? "X" : " ") + "] " + description;
+        }
+
+    }
+
+    private static class Todo extends Task {
+        Todo(String description) {
+            super(description);
+        }
+
+        @Override
+        public String toString() {
+            return "[T]" + super.toString();
+        }
+    }
+
+    private static class Deadline extends Task {
+
+        protected String by;
+
+        public Deadline(String description, String by) {
+            super(description);
+            this.by = by;
+        }
+        @Override
+        public String toString() {
+            return "[D]" + super.toString() + " (by: " + by + ")";
+        }
+    }
+
+    private static class Event extends Task {
+        protected String from;
+        protected String to;
```

Comment:
Perhaps the variable names could be improved since they are referring to a specific time. You could consider using startTime and endTime.